protos/component_metadata/component_metadata.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package mavsdk.rpc.component_metadata;
+
+import "mavsdk_options.proto";
+
+option java_package = "io.mavsdk.component_metadata";
+option java_outer_classname = "ComponentMetadataProto";
+
+// Access component metadata json definitions, such as parameters.
+service ComponentMetadataService {
+
+    /*
+     * Request metadata from a specific component
+     */
+    rpc RequestComponent(RequestComponentRequest) returns(EmptyResponse) { option (mavsdk.options.async_type) = SYNC; }
```

Comment:
Pretty sure they need to be named a very specific way:

```suggestion
    rpc RequestComponent(RequestComponentRequest) returns(RequestComponentResponse) { option (mavsdk.options.async_type) = SYNC; }
```

Reply:
From what I could tell it did not make a difference, only the Response suffix is imporant. I changed it now though.

Reply:
If I remember right, some things don't matter for C++ but might for some language bindings.